ocaml/ast.ml

```diff
@@ -0,0 +1,18 @@
+open Core
+
```

Comment:
If you want you can write something like `module L = Lexer` up here and then use `L` everywhere instead of `Lexer` to make your code shorter.